katcp/server.py

```diff
@@ -34,9 +34,12 @@
 from .sampling import format_inform_v5, format_inform_v4
 from .core import (SEC_TO_MS_FAC, MS_TO_SEC_FAC, SEC_TS_KATCP_MAJOR,
                    VERSION_CONNECT_KATCP_MAJOR, DEFAULT_KATCP_MAJOR)
-from .version import VERSION, VERSION_STR
 from .kattypes import (request, return_reply)
 
+# 'import katcp' so that we can use katcp.__version__ later
+# we cannot do this: 'from . import __version__' because __version__
+# does not exist at this stage
+import katcp
```

Comment:
The problem is that katversion insists on putting `__version__` at the end of `__init__.py` to ensure that it is authoritative. I'm not sure what will happen if you put say `__version__ = None` at the top and then replace it at the end (will `server.py` see the change?).

A separate `version.py` is the cleanest but again requires changes to katversion. I prefer the current iteration of katversion as it is simpler for most packages to just stick to `__init__.py`.

The current solution was also touted [here](http://stackoverflow.com/questions/1556387/circular-import-dependency-in-python).